### ISSUE TITLE
Expose vm services for adjusting MediaQuery defaults

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#179808]
### Use case

As doc flutter.dev/go/devtool-a11y-features (actual redirect TBD) mentioned, we need a VM services for devtool to override various devices settings

### Proposal

We should have support to adjust all settings that attributes to https://api.flutter.dev/flutter/widgets/MediaQueryData-class.html

This will need to change the default for FlutterView or PlatformDispatcher directly to better reflects the behavior when devices settings changes
